cluster-classification/script.whizzml

```diff
@@ -0,0 +1,70 @@
+;; Given a cluster, creates a batch centroid and uses the output
+;; dataset where each instance is labeled by centroid to create an
+;; ensemble (random forest), and returns the importance of each field
+;; averaged over the ensemble.
+
+;; Given a cluster, returns a batchcentroid id created from that
+;; cluster whose output dataset is the original cluster dataset
+;; appended with the centroids the instances are part of.
+
+(define (label-by-cluster cl-id) 
+  (let (cl-ds (get-in (fetch cl-id) ["dataset"]) 
+        cl-fields (get-in (fetch cl-id) ["input_fields"]))
+    (create-and-wait-batchcentroid 
+     {"cluster" cl-id 
+      "dataset" cl-ds 
+      "output_dataset" true
+      "output_fields" cl-fields})))
+
+;; Given a list of pairs, creates the equivalent map
+(define (list-to-map nested-list)
+  (let (key-list (map head nested-list)
+        value-list (map (lambda (x) (nth x 1)) nested-list))
```

Comment:
Jumping in - but since these lists have just two values, you can drop the lambda fn altogether in the new WhizzML and just use `last`.